modules/swagger-jersey2-jaxrs/src/main/java/com/wordnik/swagger/jersey/SwaggerJersey2Jaxrs.java

```diff
@@ -83,37 +73,31 @@
           }
 
           // Re-process all Bean fields and let the default swagger-jaxrs/swagger-jersey-jaxrs processors do their thing
-          List<Parameter> extracted = extensions.next().extractParameters(paramAnnotations.toArray(new Annotation[paramAnnotations.size()]),
-                                                                          paramClass,
-                                                                          ParameterUtils.isMethodArgumentAnArray(paramClass, paramType),
-                                                                          classesToSkip,
-                                                                          extensions);
+          List<Parameter> extracted =
+              extensions.next().extractParameters(paramAnnotations, paramType, typesToSkip, extensions);
 
           // since downstream processors won't know how to introspect @BeanParam, process here
-          for(Parameter param : extracted)
-            ParameterProcessor.applyAnnotations(null, param, paramClass, paramAnnotations.toArray(new Annotation[paramAnnotations.size()]), isArray);
-
-          parameters.addAll(extracted);
+          for(Parameter param : extracted) {
+            if (ParameterProcessor.applyAnnotations(null, param, paramType, paramAnnotations) != null) {
```

Comment:
The `ParameterProcessor.applyAnnotations` call may return `null` for skipped parameters, so let's skip them too.